types/errors_test.go

```diff
@@ -27,17 +32,25 @@ var errFns = []errFn{
 	ErrUnauthorized,
 	ErrInsufficientFunds,
 	ErrUnknownRequest,
-	ErrUnknownAddress,
+	ErrInvalidAddress,
 	ErrInvalidPubKey,
+	ErrUnknownAddress,
+	ErrInsufficientCoins,
+	ErrInvalidCoins,
+	ErrOutOfGas,
+	ErrMemoTooLarge,
 }
 
 func TestCodeType(t *testing.T) {
 	require.True(t, ABCICodeOK.IsOK())
 
 	for _, c := range codeTypes {
 		msg := CodeToDefaultMsg(c)
-		require.False(t, strings.HasPrefix(msg, "Unknown code"))
+		require.False(t, strings.HasPrefix(msg, "unknown code"))
```

Comment:
Is there a way we can avoid hardcoding this `unknown code`? If theres not a simple way, don't worry about it, its super minor. 

Also test message formatting: https://github.com/cosmos/cosmos-sdk/blob/develop/CONTRIBUTING.md#testing